rfcs/text/0008-threat-intel.md

```diff
@@ -23,30 +23,38 @@ Stage 1: Describe at a high level how this change affects fields. Which fieldset
 
 ### Proposed New Fields for Threat fieldset
 
-  * threat.time_first_seen
+  * threat.ioc.time_first_seen
     _The date and time when intelligence souce first reported sighting this indicator._
-  * threat.time_last_seen
+  * threat.ioc.time_last_seen
     _The date and time when intelligence source last reported sighting this indicator._
-  * threat.sightings
+  * threat.ioc.sightings
     _Number of times this indicator was observed conducting threat activity._
   * threat.type
     _Type of indicator as reprsented by Cyber Observable in STIX 2.0_
-  * threat.description
+  * threat.ioc.description
     _Describes the type of action conducted by the threat._
-  * threat.tlp
-    _Traffic Light Protocol, which dictates sharing policies_
-  * threat.classification
+threat.marking.tlp
+_Data markings represent restrictions, permissions, and other guidance for how data can be used and shared. Examples could be TLP (White, Green, Amber, Red)._
+  * threat.ioc.classification
     _Describes type of threat delivery (Hacktool etc.) and family name.
-  * threat.scanner_stats
+  * threat.ioc.scanner_stats
     _Count of Anti virus/EDR that successfully detected malicious file or URL. Sources like VirusTotal, Reversing Labs often provide these statistics._
 
-### Proposed New Values
+### Proposed New Values for Event Fieldset
 
   * event.kind:enrichment _Propose adding this value to capture outcome of this event. It could also appy to other types of contextual data such as directory services, IPAM data, asset lists._
   * event.category:threat _Proposed threat.type field would be a subcategory for this value of event.category_
   * event.type:indicator _Proposed value represents type of threat information. In future this could be extended to other STIX 2.0 Standard Data Objects like Actor, Infrastucture etc._
 
-### Existing ECS Fields For Nested Use in Threat.*
+### Using existing Event Fieldset
+ * event.reference _URL to the intelligence source_
+ * event.provider _name of intelligence provider_
+ * event.dataset _name of specific dataset from the intelligence source. Intelligence sources often provide multiple datasets - IP blocklist, File hash blocklist etc.
+ * event.severity _severity provided by threat intelligence source_
+ * event.risk_score _risk score provided by threat intelligence source_
+ * event.original _raw intelligence event_
```

Comment:
To accommodate for both matching and enrichment use cases moving these fields under `threat.*` is the right approach.

Reply:
To avoid the overwriting issue in the enriched document, we're going to add `threat.ioc.{module,provider,dataset}` as new fields under `threat.ioc.*`. We'll be adding this to the example documents as well as the `threat.yml` file.